cython/hittable_list.pyx

```diff
@@ -0,0 +1,68 @@
+
+from numpy.math cimport INFINITY
+
+cdef class HittableList(Hittable):
```

Comment:
We have a Python implementation for this one. It might be worth it to create a Cython implementation at some point though